network-api/networkapi/buyersguide/templates/buyersguide/home.html

```diff
@@ -1,3 +1,3 @@
-{% extends "../catalog.html" %}
+{% extends "./catalog.html" %}
```

Comment:
We're now using a modified version of catalog.html, so I made a copy of it. We now have 2 of them.